api/environments/identities/serializers.py

```diff
@@ -106,16 +107,15 @@ def get_feature_state_value(
         self, instance: typing.Union[FeatureState, FeatureStateModel]
     ) -> typing.Union[str, int, bool]:
         identity = self.context["identity"]
+        environment_api_key = self.context["environment_api_key"]
 
-        if isinstance(identity, Identity):
-            identity_id = identity.id
-        else:
-            identity_id = identity.django_id or identity.identity_uuid
+        environment = Environment.get_from_cache(environment_api_key)
+        hash_key = identity.get_hash_key(environment.use_mv_v2_evaluation)
```

Comment:
Ah, but I guess (based on my previous question) - we wouldn't get it from the cache then. BUT wouldn't we have the environment `select_related` on the identity already?

Reply:
The major reason was to have the same interface as the engine though, does that make sense?

Reply:
Yeah, I see that. It's quite interesting really - I guess there are pros/cons of both options. For some reason, I am still leaning towards the simpler interface here (removing the argument) but I don't think my opinion is strong enough to require a change here.